os/arch/arm/src/armv7-a/arm_mmu.c

```diff
@@ -277,6 +290,9 @@ void mmu_l1_map_regions(const struct section_mapping_s *mappings,
  ****************************************************************************/
 
 #ifndef CONFIG_ARCH_ROMPGTABLE
+#ifdef CONFIG_ARCH_CHIP_AMEBASMART
+SRAMDRAM_ONLY_TEXT_SECTION
+#endif
```

Comment:
Can you pls add this macro for `mmu_map_app_region` and ` mmu_clear_app_pgtbl` apis also, because we are using invalidate api in both these places as well. The `mmu_map_app_region` will also be called during boot up in case of loadable config.

Reply:
done~